pkg/asset/cluster/azure/azure.go

```diff
@@ -62,3 +102,7 @@ func PreTerraform(ctx context.Context, clusterID string, installConfig *installc
 	}
 	return nil
 }
+
+func sharedTag(clusterID string) (string, *string) {
+	return fmt.Sprintf("kubernetes.io_cluster.%s", clusterID), to.StringPtr("owned")
```

Comment:
It looks like the tag value is "owned" when it should be "shared". I assume we can't simply change this. How should we proceed?

Reply:
Oh, that's just confusing. The [commit message](https://github.com/openshift/installer/commit/ebd7ee6c1a997657c6290f17b9a6d6b1db4c34ae) explains why we're supplying the `owned` tag for a user-supplied resource (the resource group). That is *NOT* what we want to do with the vnets.

So I would suggest creating another similar function that really supplies a shared tag and use that to apply to the vnets. At a quick glance I think you could potentally rename this function from `sharedTag` to `ownedTag`.

Reply:
Ah thanks for that. I've taken your suggestion and also stole the comment from that commit - PTAL